modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/raft/ZonePartitionRaftListener.java

```diff
@@ -124,6 +159,24 @@ private void processWriteCommand(CommandClosure<WriteCommand> clo) {
 
             clo.result(null);
         }
+
+        // result == null means that the command either was not handled by anyone (and clo.result() is called) or
+        // that it was delegated to a table processor (which called clo.result()).
+
```

Comment:
Unnecessary blank line and I don't really like the dualism of result and closure as return value. Don't have any good ideas from scratch about how to fix it though.

Reply:
Yep, that makes sense.